src/api/generate-xlsx/index.ts

```diff
@@ -0,0 +1,69 @@
+import ExcelJS from 'exceljs';
+import mapApplicationToXLSX from './map-application-to-XLSX';
+import HEADER_COLUMNS from './header-columns';
+import { Application } from '../types';
+
+/**
+ * XLSX
+ * Generate an XLSX file with exceljs
+ * @param {Object} Application
+ * @returns {String} File path
+ */
+const XLSX = (application: Application): Promise<string> => {
+  try {
+    console.info('Generating XLSX file');
```

Comment:
To be done later, is it maybe worth putting `for: ${application.id}` at the end for nice logging?

Reply:
Great idea i'll put this into the next PR